### ISSUE TITLE
Calling toNumber() on a unitless unit (math.Unit instance) causes an error

### ISSUE THREAD

**Issue by pedroasad** [josdejong/mathjs#1808]
Using `mathjs@6.6.2`

The `math.unit()` constructor allows for unitless quantities, that is, specifying a value without a unit name, like `math.unit(3)`. However, objects instantiated like this cause an error when the `toNumber()` metho is called.

Minimal steps to reproduce (no Node.js console):

```node
> const math = require('mathjs')
> const x = math.unit(3)
> x.toNumber()  // expected result is 3
TypeError: Cannot read property 'prefix' of undefined
    at Unit.toNumeric (<working_dir>/node_modules/mathjs/lib/type/unit/Unit.js:914:61)
    at Unit.toNumber (<working_dir>/node_modules/mathjs/lib/type/unit/Unit.js:891:26)
```

The error seems to happen on [`src/type/unit/Unit.js:886`](../tree/v6.6.2/src/type/unit/Unit.js).

**Comment by josdejong** [comment]
Thanks for reporting Pedro. I've created a PR with a fix in #1811. @ericman314 can you have a look please?

**Comment by josdejong** [comment]
This issue is fixed now in `mathjs@6.6.3`
